tests/integration/conftest.py

```diff
@@ -11,3 +13,15 @@ def opensearch_sysctl_settings():
     subprocess.run(["sudo", "sysctl", "-w", "vm.swappiness=0"])
     subprocess.run(["sudo", "sysctl", "-w", "vm.max_map_count=262144"])
     subprocess.run(["sudo", "sysctl", "-w", "net.ipv4.tcp_retries2=5"])
+
+
+@pytest.fixture(scope="module")
+async def application_charm(ops_test: OpsTest):
+    """Build the application charm."""
+    test_charm_path = "tests/integration/application-charm"
+    shutil.copyfile(
+        "lib/charms/data_platform_libs/v0/data_interfaces.py",
+        f"{test_charm_path}/lib/charms/data_platform_libs/v0/data_interfaces.py",
+    )
+    # charm = await ops_test.build_charm(test_charm_path)
+    # return charm
```

Comment:
Aiming to use the above "usual" mechanism, I keep getting the following error: https://github.com/canonical/opensearch-dashboards-operator/actions/runs/9413656493/job/25931418984#step:21:183

(Both locally and on the pipelines.)
I spent non-negligable time on trying to resolve this, with no success. I can't tell if I may be missing something obvious, or there may be a library mismatch or so...? I checked against other charms using the usual fixture "way", they seem to match with libs I'm using in the 1st place...

If anyone can see what I may be missing, please let me know.